haiku-apps/projectconceptor/projectconceptor-1.0.0.recipe

```diff
@@ -0,0 +1,59 @@
+SUMMARY="A Haiku diagramming application like Microsoft Visio"
+DESCRIPTION="
+	ProjectConceptor is a modular application for Haiku and BeOS designed to \
+	view, edit and process information which can be represented by a graph \
+	structure.
+	"
+APP="ProjectConceptor"
+HOMEPAGE="https://github.com/Paradoxianer/$APP"
+COPYRIGHT="
+	2017 Paradoxianer
+	2000-2002 Lee Thomason
+	1991-2000 Be Incorporated
+	"
+LICENSE="
+	BSD (3-clause)
+	MIT
```

Comment:
We have to ship a copy of [Open Tracker License][0] also

[0]: https://github.com/Paradoxianer/ProjectConceptor/blob/dfef4b25fbd3dc620e144fef66a0879c4442a8f1/src/app/Tools/ColorTools.h#L4

Reply:
I had already seen Open Tracker in the ColorTools code, but it's not a valid license as far as `haikuporter` is concerned.

Reply:
You can add custom licenses by making a folder in the same place as your recipe called "licenses". In there, add a file called "Open Tracker License" with the content of the said [license](https://github.com/Paradoxianer/ProjectConceptor/blob/dfef4b25fbd3dc620e144fef66a0879c4442a8f1/src/app/Tools/ColorTools.h#L4), then it would become vaild for `haikuporter`.

Reply:
Thanks it works.

Reply:
```diff
-COPYRIGHT="
-	2017 Paradoxianer
+COPYRIGHT="2017 Paradoxianer
 	2000-2002 Lee Thomason
-	1991-2000 Be Incorporated
-	"
+	1991-2000 Be Incorporated"
-LICENSE="
-	BSD (3-clause)
+LICENSE="BSD (3-clause)
 	MIT
 	Open Tracker License
-	Zlib
-	"
+	Zlib"
```
The recommended style for "multi-line" `COPYRIGHT` and `LICENSE` is to have the first author or license in the same line as the name of the parameter, right after the `="`, and the closing quotes right after the last author or license.

I just did some metrics on `COPYRIGHT` and `LICENSES`. The following tables **only** take into account thoses recipes which have **2 or more lines** in these variables.

Number of recipes with multi-line `COPYRIGHT` | 553
--- | ---
Number of recipes using the _recommended_ style for `COPYRIGHT` | 493
Number of recipes using the _alternate_ style for `COPYRIGHT` | 60


Number of recipes with multi-line `LICENSE` | 232
--- | ---
Number of recipes using the _recommended_ style for `LICENSE` | 170
Number of recipes using the _alternate_ style for `LICENSE` | 62

Since this PR is for adding a new recipe, it would be nice to follow the trend :)
Thanks!

Hmm, if you are away from your Haiku system then I can push these small changes for you if you agree :)

(I've already updated the wiki with an [example of a recipe having a multi-line](https://github.com/haikuports/haikuports/wiki/HaikuPorter-BuildRecipes#a-simple-example-using-the-recommended-style-for-a-multi-line-copyright) `COPYRIGHT`.)

Reply:
Thanks! Will fix it soon.

Reply:
Maybe we can use this recipe as a simple example to avoid having two simple examples with the recommended style?